src/murmur/MurmurIce.cpp

```diff
@@ -57,6 +59,20 @@ static std::string iceString(const QString &s) {
 	return iceRemoveNul(u8(s));
 }
 
+/// iceStringBase64 converts the Ice-safe string in std::string
+/// (assumed to be the output of iceString) to base64 using the
```

Comment:
This is not correct, is it? The context does not have to be changed/cleaned *because* we base64 it instead to preserve it. (The change below also changes the call to not include the call to iceString)